src/com/google/enterprise/adaptor/prebuilt/FilterMimetypes.java

```diff
@@ -131,14 +131,14 @@ public void transform(Metadata metadata, Map<String, String> params) {
     ct = ct.trim().toLowerCase();
     String decision = lookupDecision(ct);
     if (null != decision) {
-      params.put("Transmission-Decision", decision);
+      if (!decision.equals(TransmissionDecision.AS_IS.toString())) {
```

Comment:
Please add note that we store AS_IS decisions for short circuiting search, however we do provide the decision back because we don't want to override another filters exclusion (that's the reason right?).